mhs-reference-implementation/mhs/routing/tests/test_sds.py

```diff
@@ -0,0 +1,96 @@
+from unittest import TestCase
+import mhs.routing.sds as sds
+import mhs.routing.tests.ldap_mocks as mocks
+from utilities.test_utilities import async_test
+import mhs.routing.routing_exception as re
```

Comment:
Minor: `re` is the name of the regex stdlib module, so it's a little confusing to use that name.

Reply:
Updated to just use the module name